ros2topic/ros2topic/verb/echo.py

```diff
@@ -97,8 +99,10 @@ def represent_ordereddict(dumper, data):
 def subscriber(node, topic_name, message_type, callback):
     if message_type is None:
         topic_names_and_types = get_topic_names_and_types(node=node)
+        expanded_name = expand_topic_name(topic_name, node.get_name(), node.get_namespace())
+        validate_full_topic_name(expanded_name, is_service=False)
```

Comment:
Nit: `False` being the default you shouldn't need to have to pass the `is_service` keyword

Reply:
Good call.  Will fix that before merging.